lib/genericCommand.js

```diff
@@ -115,11 +115,10 @@ module.exports = function(cmd) {
     if (typeof postCmd === 'function') {
       // Apply a useful scope in these functions
       var post = function(params, cb) {
-        postCmd.apply(cmd, [params, cb]);
+        postCmd.apply(cmd, [argv, params, cb]);
```

Comment:
We found that this error is the master. The JIRA created to solve it is: https://github.com/feedhenry/fh-fhc/pull/363. And it will be solved after merge this.